src/pages/dataElements/form/DataElementFormFields.tsx

```diff
@@ -10,157 +8,71 @@ import {
-                        required
-                        inputWidth="400px"
-                        label={i18n.t('{{fieldLabel}} (required)', {
-                            fieldLabel: i18n.t('Name'),
-                        })}
-                        name="name"
-                        helpText={i18n.t(
-                            'A data element name should be concise and easy to recognize.'
-                        )}
-                        validate={(value) => (!value ? 'Required' : undefined)}
-                    />
-                </StandardFormField>
-
-                <StandardFormField>
-                    <FieldRFF
-                        required
-                        component={InputFieldFF}
-                        dataTest="dataelementsformfields-shortname"
-                        validate={(value) => (!value ? 'Required' : undefined)}
-                        inputWidth="400px"
-                        name="shortName"
-                        label={i18n.t('{{fieldLabel}} (required)', {
-                            fieldLabel: i18n.t('Short name'),
-                        })}
-                        helpText={i18n.t(
-                            'Often used in reports where space is limited'
-                        )}
-                    />
-                </StandardFormField>
-
-                <StandardFormField>
-                    <FieldRFF
-                        component={InputFieldFF}
-                        dataTest="dataelementsformfields-code"
-                        inputWidth="150px"
-                        name="code"
-                        label={i18n.t('Code')}
-                    />
-                </StandardFormField>
-
-                <StandardFormField>
-                    <FieldRFF
-                        component={TextAreaFieldFF}
-                        dataTest="dataelementsformfields-description"
-                        inputWidth="400px"
-                        name="description"
-                        label={i18n.t('Description')}
-                        helpText={i18n.t(
-                            "Explain the purpose of this data element and how it's measured."
-                        )}
-                    />
-                </StandardFormField>
-
-                <StandardFormField>
-                    <FieldRFF
-                        component={InputFieldFF}
-                        dataTest="dataelementsformfields-url"
-                        inputWidth="400px"
-                        name="url"
-                        label={i18n.t('Url')}
-                        helpText={i18n.t(
-                            'A web link that provides extra information'
-                        )}
-                    />
+                    <NameField />
+                </StandardFormField>
+
+                <StandardFormField>
+                    <ShortNameField />
+                </StandardFormField>
+
```

Comment:
We should move the `FormName` here. See https://docs.google.com/spreadsheets/d/1WLEtF81gYoQHhKcdO7c-ZioyO9muqLWwIGeeqZRqvIM/edit#gid=461568956